### DISCUSSION TITLE
posts/howto-convert-dyn2base/

### DISCUSSION THREAD

**Discussion by giscus** [Herve-M/blog#8] [BETA Annoucements]
# posts/howto-convert-dyn2base/

Welcome to N(H)erve'd Powered Chronicles—a quirky blend of code, creativity, and (maybe too much) caffeine. Dive into a world where tech meets humor, and every line of code comes with a side of adventure… or possibly disappointment!

https://beta.nherve.dev/posts/howto-convert-dyn2base/

**Comment by Herve-M** [comment]
This is a test!
